h/sql_tasks/tasks/report/create_from_scratch/03_annotation_group_counts/01_create_view.sql

```diff
@@ -6,7 +6,9 @@ CREATE MATERIALIZED VIEW report.annotation_group_counts AS (
         DATE_TRUNC('week', created)::date AS created_week,
         authority_id,
         group_id,
-        COUNT(1)::integer AS count
+        COUNT(1)::INTEGER AS count,
+        (COUNT(1) FILTER (WHERE shared IS TRUE))::INTEGER AS shared,
+        (COUNT(1) FILTER (WHERE ARRAY_LENGTH(parent_uuids, 1) > 0))::INTEGER AS replies
```

Comment:
I only found out you could do this relatively recently, and I think it might have simplified some of our other super long queries which count multiple ways at once.